cli/src/pcluster/models/cluster.py

```diff
@@ -535,7 +535,9 @@ def _render_and_upload_scheduler_plugin_template(self):
             )
             template = environment.from_string(file_content)
             rendered_template = template.render(
-                cluster_configuration=parse_config(self.source_config_text), cluster_name=self.name
+                cluster_configuration=parse_config(self.source_config_text),
```

Comment:
Can we extend `test_render_and_upload_scheduler_plugin_template` to test that jinja rendering is properly performed by trying to use both the hash function and also the passed parameters?

Reply:
Added return value for get_instance_types_data method in test. Shall we add some jinja template examples here?